File: tests/OuterClassTest.php
<?php
namespace PHPJava\Tests;

class OuterClassTest extends Base
{
    protected $fixtures = [
        'OuterClassTestOuterClass',
        'OuterClassTest',
    ];

    public function testCallMain()
    {
        ob_start();
        $calculatedValue = static::$initiatedJavaClasses['OuterClassTest']
            ->getInvoker()
            ->getStatic()
            ->getMethods()
            ->call(
                'main',
                []
            );
        $result = ob_get_clean();

        $this->assertEquals(
            'Called Static Method AND Called Dynamic Method',
            $result
        );
    }
}

File: tests/fixtures/java/OuterClassTest.java
class OuterClassTest
{
    public static void main(String[] args)
    {
        System.out.print(OuterClassTestOuterClass.staticHelloWorld());
        System.out.print(" AND ");
        System.out.print((new OuterClassTestOuterClass()).dynamicHelloWorld());
    }
}
